woocommerce/woocommerce
# Update payment partners 2024 q1

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/Automattic/woocommerce.com/issues/19237


1. Removed ZipCo from all countries
2. Removed Eway from APAC (all countries since Eway was only added to APAC countries)
3. Added new partner `AirWallex` ,  located right below `Paypal` for EU+UK, located below `Stripe` and above `Paypal` for APAC
    - Title: `Airwallex Payments`
    - Description: `Boost international sales and save on FX fees. Accept 60+ local payment methods including Apple Pay and Google Pay.`
    - Countries: `[ 'GB', 'AT', 'BE', 'EE', 'FR', 'DE', 'GR', 'IE', 'IT', 'NL', 'PL', 'PT', 'AU', 'NZ', 'HK', 'SG', 'CN' ]`
4. Fixed lint `Short array syntax is not allowed` errors.

- ~Re-order `Mercado Pago` in LATAM. It should now be above PayPal~  **(It's already above PayPal)**
 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Start with a fresh install
1. Go to core profiler
1. Choose `UK` as store country
1. Complete core profiler without installing any plugins
1. Go to `WooCommerce > Settings > Advanced > WooCommerce.com` > **turn off** "Show Suggestions" checkbox
1. Go to `WooCommerce -> Home` and choose the `Set up payments` task.
1. Confirm it shows Stripe, PayPal Payments, Airwallex Payments, and Mollie in order under the "Other payment providers" section.
1. Confirm you can install Airwallex Payments via payment task
1.  Change store country to Australia
1. Go to `Set up payments` task.
1. Confirm it shows Stripe, Airwallex Payments and PayPal Payments in order under "Other payment providers" section. (No Eway)
1. Install WCpay
1. Go to `Tools > WCA Test Helper > REST API FIlters` (/wp-admin/tools.php?page=woocommerce-admin-test-helper)

```
Click new filters:

- Endpoint: /wc/v3/payment_gateways
- Dot Notation: 0.id
- Replacement: woocommerce_payments

- Endpoint: /wc/v3/payment_gateways
- Dot Notation: 0.needs_setup
- Replacement:false

- Endpoint: /wc/v3/payment_gateways
- Dot Notation: 0.enabled
- Replacement: true
```

15. You should see the `Additional payment options` section as follows (no ZipCo)

![Screenshot 2024-01-11 at 20 21 50](https://github.com/Automattic/woocommerce.com/assets/4344253/04dd06a6-4af7-485d-bb96-5188827a7aac)

16. Please test around



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Update onboarding payment partners 

#### Comment 

</details>
